ia/src/protocole.py

```
@@ -139,10 +138,12 @@ def connect_client(server_address, team_name) -> int:
     send_message(client, team_name)
 
     game_data = client["socket"].recv(1024).decode()
+    if game_data == "ko\n":
+        print("Unknown team name or team is full")
+        return 0
+  
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)